01-report.js

```diff
@@ -5,21 +5,18 @@
  * @returns The sales report for the last 12 months
  */
 export function getLast12MonthReport(database) {
-    // TODO: This method does not work, I don't know why.
-    // Please have a look, I really don't see anything wrong.
-    // Note: This one is a bit hard, you will need to use the debugger to understand what is going on, or use the console.log.
-    // Please don't get stuck too long on this one, ask for help if you need it.
-    // Also, reading the doc might be helpful :D
     const today = new Date();
     const d = new Date();
-    const year = d.getFullYear();
+    //Hack to set the day to be the 2nd of the month on normal years, and the first on leap yars
```

Comment:
```suggestion
    //Hack to set the day to be the 2nd of the month to avoid issues with leap years (february 29th doesn't exist one year before)
```